src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -283,10 +281,6 @@ const PaymentInformationContainer = connect(
   mapDispatchToProps,
 )(PaymentInformation);
 
-const noop = () => null;
-
-export default (featureFlags.directDeposit
-  ? PaymentInformationContainer
-  : noop);
+export default PaymentInformationContainer;
```

Comment:
We are now relying on the parent component to conditionally show/hide this PaymentInformation component.